packages/language-server/src/plugins/html/features/astro-attributes.ts

```diff
@@ -70,6 +70,16 @@ export const astroAttributes = newHTMLDataProvider('astro-attributes', {
 						},
 					],
 				},
+				{
+					name: 'global',
+					description: 'Opts-out of automatic CSS scoping, all contents will be available globally',
```

Comment:
```suggestion
					description: 'Opts-out of automatic CSS scoping, all contents will be available globally',
					valueSet: 'v',
```

By adding the valueSet  parameter and setting it to `v`, the completion will be nicer (`global` instead of `global=""`)